### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, I added Views associated with the Models and Controllers.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+        postsCounter: 1
+      },
+      {
+        name: 'Simon',
+        photo: 'https://images-na.ssl-images-amazon.com/images/I/41vMBIeF-zL._SX384_BO1,204,203,200_.jpg',
+        bio: 'Born at a very young age.',
+        postsCounter: 2
+      }
+    ]
+  end
```
Comment (AshakaE) [inline app/controllers/users_controller.rb]: This is bad practice, you're expected to have this data in your DB so the command `@users = User.all` renders all the users from the DB to the action view. You create seed data for app testing.

- [ ] Please update code

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+      {
+        title: 'post 2',
+        text: 'this is post content'
+      },
+      {
+        title: 'post 3',
+        text: 'this is post content'
+      }
+    ]
   end
```
Comment (AshakaE) [inline app/controllers/users_controller.rb]: - [ ] Please seed your test data and use the right command to retrieve data from the DB

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,30 @@
...
+<h4> <%= post[:title]%> </h4>
+<p> <%= post[:text]%> </p>
+<div class="d-flex justify-content-end">
+<span>Comments: x</span>
+<span>Likes: y </span>
+</div>
+</div>
+<% end %>
+<div class=" container col-12 text-center">
+<button class="border rounded">See all posts</button>
```
Comment (AshakaE) [inline app/views/users/show.html.erb]: Button does not work for `see all posts`

- [ ] Please use the right tag so when a user clicks `see all posts` they can view it.

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -1 +1,30 @@
...
+  <p><%= @user[:bio] %></p>
+  </div>
+</div>
+<% @posts.each do |post| %>
+<div class="container border rounded my-3">
+<h4> <%= post[:title]%> </h4>
+<p> <%= post[:text]%> </p>
+<div class="d-flex justify-content-end">
+<span>Comments: x</span>
+<span>Likes: y </span>
```
Comment (AshakaE) [inline app/views/users/show.html.erb]: - [ ] Please render the values for comments and likes

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @denisdiaconu  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**